src/main/kotlin/org/jetbrains/research/testspark/display/generatedTests/TopButtonsPanelBuilder.kt

```diff
@@ -107,21 +107,33 @@ class TopButtonsPanelBuilder {
             testCasePanelFactory.addTask(tasks)
         }
         // run tasks one after each other
-        executeTasks(project, tasks)
+        executeTasks(project, tasks, runAllButton, generatedTestsTabData)
```

Comment:
You do not need to provide `runAllButton` into `executeTasks`, as this button is already a member field of the current class:
```kotlin
class TopButtonsPanelBuilder {
    private var runAllButton: JButton = createRunAllTestButton()
```